.github/workflows/ci.yml

```
@@ -58,16 +56,33 @@ jobs:
       - run: python -m pip install -r requirements.txt
       - run: ${{ matrix.lint-command }}
 
-  readme:
+  build-and-validate:
     runs-on: ubuntu-latest
     steps:
-      - run: sudo apt install -y gettext
+      - run: |
+          sudo apt update
+          sudo apt install -y gettext
       - uses: actions/setup-python@v5.6.0
       - uses: actions/checkout@v4
-      - name: Install Python dependencies
-        run: python -m pip install --upgrade pip setuptools wheel twine readme-renderer
-      - run: python setup.py sdist bdist_wheel
-      - run: python -m twine check dist/*
+      - name: Install uv and readme-renderer
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install uv
+          uv venv
+          source .venv/bin/activate
+          uv pip install readme-renderer
+      - name: Clean previous builds
+        run: rm -rf build dist *.egg-info
```

You are on a clean container, no need to remove anything.
```suggestion
      - name: Clean previous builds
        run: rm -rf build dist *.egg-info
```